packages/react-components/tests/dispensers/dispenser-marker.spec.tsx

```diff
@@ -2,19 +2,27 @@ import { render } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import React from 'react';
 import { DispenserMarker } from '../../lib';
+import { mockOnClick } from '../test-utils';
+
+it('triggers onClick callback when button is clicked', () => {
+  const handler = mockOnClick();
+  spyOn(handler, 'onClick');
```

Comment:
Not sure how jasmine spy works, can we just create a spy directly instead of using an object? https://stackoverflow.com/questions/9510148/using-jasmine-to-spy-on-a-function-without-an-object.

Reply:
Yeah we can. I'll create it this way `const mockOnClick = jasmine.createSpy();`